filelists/CIFAR-FS/cifar_fs_preprocessing.py

```diff
@@ -0,0 +1,490 @@
+# Dataloader of Gidaris & Komodakis, CVPR 2018
+# Adapted from:
+# https://github.com/gidariss/FewShotWithoutForgetting/blob/master/dataloader.py
+from __future__ import print_function
+
+import os
+import os.path
+import numpy as np
+import random
+import pickle
+
+import torch
+import torch.utils.data as data
+import torchvision.transforms as transforms
+import torchnet as tnt
+
+
+from PIL import Image
+
+# Set the appropriate paths of the datasets here.
+_CIFAR_FS_DATASET_DIR = '/home/azmkuu/Desktop/EPFL/OPTML/CloserLookFewShot/filelists/CIFAR-FS'
```

Comment:
An automatic download would also make this unnecessary. The other downloaders save the data in `filelists`